components/Common/Search/States/WithSearchBox.tsx

```diff
@@ -78,15 +80,55 @@ export const WithSearchBox: FC<SearchBoxProps> = ({ onClose }) => {
     [searchTerm, selectedFacet]
   );
 
+  useKeyboardCommands(cmd => {
+    if (searchError || !searchResults || searchResults.count <= 0) return;
+
+    switch (true) {
+      case cmd === 'down' && selectedResult == null:
```

Comment:
I've never seen boolean algebra done this way with a switch statement. Kinda odd, but neat,